### DISCUSSION TITLE
oraganization

### DISCUSSION THREAD

**Discussion by MohmdAshraf22** [community/community#36406] [New to GitHub]
I turned my account into an organization
How do I get it back ?

**Comment by DSDmark** [comment]
Hey @MohmdAshraf22,

According to that official **GitHub** documentation 

> It's not possible to convert an organization into a personal account, but you can create a new personal account and transfer the organization's repositories to it.
[**GitHub docs**](https://docs.github.com/en/organizations/managing-organization-settings/converting-an-organization-into-a-user)

I think you should delete that **organization** and create new account, here is details:-

[**Deleting Organization**](https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account)

> Note: After an account is deleted, the username at the time of deletion becomes unavailable for reuse for 90 days. To reuse an organization's username immediately, you must change the username before you delete the organization.

**Comment by jfullstackdev** [comment]
that's quite tricky, maybe try Support first but there is no assurance

https://support.github.com/contact/account

maybe they will restore them for you, releasing usernames are tricky, we had several discussions of that in the past, also, the email will be a possible problem too, say, if you delete the Org and was not able to retrieve the email as being used by the Org, that will lose all of the commits associated with it and the badges

but if you don't have much activity in that old account, you can simple keep your new account and never make the same mistake again, anyways, creating Orgs from your personal accounts are harmless,

**Comment by ossanna16** [comment]
Hey there @MohmdAshraf22 and thank you for asking a good question! I'm glad you found our community 🙂
